### PR TITLE
Use Marie's full name in the README file

### PR DESCRIPTION
**Pull request by wolfedale**
I think full last name looks better :-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,4 +57,4 @@ There are many ways to get involved with Curiefense.
 
 <br>
 
-This project is named after the famous scientist [Marie Curie](https://www.curiefense.io/marie-curie). It began in intensive work sessions at Malakoff France, close to her home and laboratory in the outskirts of Paris, and is being released on her birthday (November 7th).
+This project is named after the famous scientist [Marie Skłodowska-Curie](https://www.curiefense.io/marie-curie). It began in intensive work sessions at Malakoff France, close to her home and laboratory in the outskirts of Paris, and is being released on her birthday (November 7th).
```
Comment (jdorfman) [inline README.md:60]: In this context, I think you are right. Wikipedia has it at "Marie Salomea Skłodowska Curie". Could you change it to that and I'll merge.

Comment (wolfedale) [reply README.md]: Just changed it, thanks! :-)

**Comment by flaper87** [comment]
@wolfedale could you sign the commit so that the DCO check can pass? :) 

Here's some info: https://github.com/curiefense/curiefense/pull/562/checks?check_run_id=3359059053

**Comment by jdorfman** [comment]
@wolfedale TL;DR

Run:

`git rebase HEAD~2 --signoff`

Now your commits will have your sign off. Next run

`git push --force-with-lease origin main`

**Comment by wolfedale** [comment]
Does it looks fine now?

**Review by jdorfman — CHANGES_REQUESTED** [review]
@wolfedale need you to remove that hyphen =)

Should be: "Marie Salomea Skłodowska Curie"

not: "Marie Salomea Skłodowska-Curie"

**Review by flaper87 — APPROVED** [review]

**Review by jdorfman — APPROVED** [review]

**Pull request merged**